org.openstream.music.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: org.openstream.music
```

Comment:
Do you own or control http://openstream.org/? It redirects to the domain provider

Reply:
The website from upstream readme is https://openstreamorg.github.io/

So I guess the app-id should be `io.github.openstreamorg.openstreammusic`

Reply:
Just to clarify. This is a requirement / blocker.